docs/.ci/git-config.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+set -e
+set -o pipefail
+
+SSH_FILE=
+while test $# -gt 0; do
+    case "$1" in
+        --ssh-file | -s)
+            shift
+            SSH_FILE=$1
+            shift
+            ;;
+        *)
+            echo "$1 is not a recognized flag!"
+            exit 1;
+            ;;
+    esac
+done
+readonly SSH_FILE
+
+cd /home/prow/go/src/github.com/kyma-project/website
```

Comment:
What is the purpose of that traversing?

Anyway, how about moving this script to `test-infra` repository? It a process of configuration `git` for `kyma-bot`, so it can be used also for other pipelines. @pkosiec What do you think?

Reply:
:+1: Great idea